StatePrinter.Tests/ExamplesForDocumentation/ExampleListAndArrays.cs

```diff
@@ -41,23 +41,25 @@ public void ExampleListAndArraysAlternative()
             vendorManager.AddVendor(JobType.JobType3, added3);
 
             var expected = @"new VendorAllocation[]()
-[0] = new VendorAllocation()
 {
-    Allocation = 100
-    Price = 20
-    Share = 20
-}
-[1] = new VendorAllocation()
-{
-    Allocation = 120
-    Price = 550
-    Share = 30
-}
-[2] = new VendorAllocation()
-{
-    Allocation = 880
-    Price = 11
-    Share = 50
+    new VendorAllocation()
```

Comment:
I thought it would be nice to produce more C#-like code, that's why I removed the indices.

Since you prefer to have them, I put them back. I don't have strong feelings either way and I see how they can be useful.